src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md

```diff
@@ -0,0 +1,28 @@
+---
+title: ‘View Netlify logs data with New Relic'
+summary: 'Add observability to your Jamstack application with the New Relic Netlify plugin and quickstart'
+releaseDate: '2022-03-30'
+learnMoreLink: ' 
+getStartedLink: '
+---
+Built for enterprise customers who want visibility of log data without the manual work. New Relics integration provides developers to import logs into the log management solution of their choice for troubleshooting and performance monitoring, or maintain them for historical purposes. Learn more here.
```

Comment:
@maicadk, here are some questions/suggestions: 

How about inserting the subject like this: "New Relics Netlify integration is built for enterprise customers who want visibility of log data without the manual work. This integration allows developers to import logs into the log management solution of their choice for troubleshooting and performance monitoring or storing them for historical purposes. Learn more here [DO YOU HAVE A LINK?](https://some-website).

Reply:
https://newrelic.com/blog/nerdlog/monitoring-netlify-sites